src/components/Movie-Detail/MovieDetail.jsx

```diff
@@ -0,0 +1,102 @@
+import { useState, useEffect } from "react";
+import axios from "axios";
+import { useParams } from "react-router-dom";
+
+function MovieDetail() {
+  const { id } = useParams();
+  const [movie, setMovie] = useState(null);
+  const [relatedMovies, setRelatedMovies] = useState([]);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(null);
+
+  const BASE_URL = import.meta.env.VITE_NODE_URL;
+
+  useEffect(() => {
+    setLoading(true);
+    async function getMovieData() {
+      try {
+       
+        const movieData = await axios.get(`${BASE_URL}/movie-data`);
```

Comment:
You can even use fetch api. It does not need any import

Reply:
ok